### PR TITLE
Add Shrink Clause on the Modify Table Partition PL/SQL

### PR DESCRIPTION
**Pull request by ilyasahsan123**

### CODE REVIEW

**Comment by ilyasahsan123** [comment]
Hi @KvanTTT, Kindly need your review for this PR. Thanks

File: sql/plsql/PlSqlParser.g4 [hunk 1]
```diff
@@ -4744,7 +4744,7 @@ modify_table_partition
     : MODIFY (
         PARTITION partition_name ((ADD | DROP) list_values_clause)? (ADD range_subpartition_desc)? (
             REBUILD? UNUSABLE LOCAL INDEXES
-        )?
+        )? (shrink_clause)?
```
Comment (KvanTTT) [inline sql/plsql/PlSqlParser.g4:4747]: Please remove redundant parentheses:

```suggestion
        )? shrink_clause?
```

**Review by KvanTTT — CHANGES_REQUESTED** [review]
Thanks for the contribtion and welcome to ANTLR grammars repository!

File: sql/plsql/PlSqlParser.g4 (hunk 1 above)
Comment (ilyasahsan123) [reply sql/plsql/PlSqlParser.g4]: Thanks for your suggestion. I have addressed it.

**Pull request merged**